homeassistant/components/uptimerobot/sensor.py

```diff
@@ -44,7 +48,9 @@ async def async_setup_entry(
                 key=str(monitor.id),
```

Comment:
Side note: We should preferably break out the description to a constant and just set `key` to eg `"monitor_status`. The entity name can be set separately with the monitor.